queries/analytics/event/saveEvent.js

```diff
@@ -33,17 +33,19 @@ async function relationalQuery(website_id, { session_id, url, event_name, event_
 
 async function clickhouseQuery(
   website_id,
-  { event_uuid, session_uuid, url, event_name, event_data },
+  { session: { country, ...sessionArgs }, event_uuid, url, event_name, event_data },
 ) {
   const { getDateFormat, sendMessage } = kafka;
+
   const params = {
     event_uuid,
     website_id,
-    session_uuid,
     created_at: getDateFormat(new Date()),
     url: url?.substring(0, URL_LENGTH),
     event_name: event_name?.substring(0, EVENT_NAME_LENGTH),
     event_data: JSON.stringify(event_data),
+    ...sessionArgs,
```

Comment:
What are sessionArgs?

Reply:
we pass the entire session object now, where sessionArgs is the rest of the session data ( hostname, browser, os, device, language).  Country is the only property that needs special care.